.github/workflows/test-pull-request.yml

```diff
@@ -32,11 +32,23 @@ jobs:
       - name: Build for tvOS
         run: xcodebuild -scheme OpenPass -destination "generic/platform=tvOS"
 
+      - name: Build ObjC for iOS
```

Comment:
Not a blocker for this PR, but is it time to introduce Fastlane into this project?